src/widget.rs

```diff
@@ -1172,6 +1172,18 @@ impl Children {
         self
     }
 
+    /// Creates [`Stack::columns`] from self.
```

Comment:
I'm not sure I want these here. Is there an argument for putting something in a "single column"? I added into_rows/into_columns to children, but I avoided putting them here because I didn't think a single-item stack was useful.

Reply:
oh you have those... I just didn't find them then this can go :D